2019-11-19 16:21:24,882 - telegram.ext.updater - INFO - Received signal 2 (SIGINT), stopping...
2019-11-20 12:24:13,569 - telegram.ext.updater - INFO - Received signal 15 (SIGTERM), stopping...
2019-11-21 10:29:43,612 - __main__ - WARNING - Update "{'update_id': 62101487, 'message': {'message_id': 214, 'date': 1574332183, 'chat': {'id': 1009602212, 'type': 'private', 'username': 'LordBenAtkinson', 'first_name': 'Ben', 'last_name': 'Atkinson'}, 'text': '/deactivate', 'entities': [{'type': 'bot_command', 'offset': 0, 'length': 11}], 'caption_entities': [], 'photo': [], 'new_chat_members': [], 'new_chat_photo': [], 'delete_chat_photo': False, 'group_chat_created': False, 'supergroup_chat_created': False, 'channel_chat_created': False, 'from': {'id': 1009602212, 'first_name': 'Ben', 'is_bot': False, 'last_name': 'Atkinson', 'username': 'LordBenAtkinson', 'language_code': 'en'}}, '_effective_user': {'id': 1009602212, 'first_name': 'Ben', 'is_bot': False, 'last_name': 'Atkinson', 'username': 'LordBenAtkinson', 'language_code': 'en'}, '_effective_chat': {'id': 1009602212, 'type': 'private', 'username': 'LordBenAtkinson', 'first_name': 'Ben', 'last_name': 'Atkinson'}, '_effective_message': {'message_id': 214, 'date': 1574332183, 'chat': {'id': 1009602212, 'type': 'private', 'username': 'LordBenAtkinson', 'first_name': 'Ben', 'last_name': 'Atkinson'}, 'text': '/deactivate', 'entities': [{'type': 'bot_command', 'offset': 0, 'length': 11}], 'caption_entities': [], 'photo': [], 'new_chat_members': [], 'new_chat_photo': [], 'delete_chat_photo': False, 'group_chat_created': False, 'supergroup_chat_created': False, 'channel_chat_created': False, 'from': {'id': 1009602212, 'first_name': 'Ben', 'is_bot': False, 'last_name': 'Atkinson', 'username': 'LordBenAtkinson', 'language_code': 'en'}}}" caused error "'Updater' object has no attribute 'message'"
2019-11-21 10:29:47,662 - telegram.ext.updater - INFO - Received signal 2 (SIGINT), stopping...
2019-11-21 10:35:44,002 - __main__ - WARNING - Update "{'update_id': 62101489, 'message': {'message_id': 219, 'date': 1574332543, 'chat': {'id': 1009602212, 'type': 'private', 'username': 'LordBenAtkinson', 'first_name': 'Ben', 'last_name': 'Atkinson'}, 'text': '/hello', 'entities': [{'type': 'bot_command', 'offset': 0, 'length': 6}], 'caption_entities': [], 'photo': [], 'new_chat_members': [], 'new_chat_photo': [], 'delete_chat_photo': False, 'group_chat_created': False, 'supergroup_chat_created': False, 'channel_chat_created': False, 'from': {'id': 1009602212, 'first_name': 'Ben', 'is_bot': False, 'last_name': 'Atkinson', 'username': 'LordBenAtkinson', 'language_code': 'en'}}, '_effective_user': {'id': 1009602212, 'first_name': 'Ben', 'is_bot': False, 'last_name': 'Atkinson', 'username': 'LordBenAtkinson', 'language_code': 'en'}, '_effective_chat': {'id': 1009602212, 'type': 'private', 'username': 'LordBenAtkinson', 'first_name': 'Ben', 'last_name': 'Atkinson'}, '_effective_message': {'message_id': 219, 'date': 1574332543, 'chat': {'id': 1009602212, 'type': 'private', 'username': 'LordBenAtkinson', 'first_name': 'Ben', 'last_name': 'Atkinson'}, 'text': '/hello', 'entities': [{'type': 'bot_command', 'offset': 0, 'length': 6}], 'caption_entities': [], 'photo': [], 'new_chat_members': [], 'new_chat_photo': [], 'delete_chat_photo': False, 'group_chat_created': False, 'supergroup_chat_created': False, 'channel_chat_created': False, 'from': {'id': 1009602212, 'first_name': 'Ben', 'is_bot': False, 'last_name': 'Atkinson', 'username': 'LordBenAtkinson', 'language_code': 'en'}}}" caused error "is_group_chat() missing 1 required positional argument: 'self'"
2019-11-21 10:35:50,681 - telegram.ext.updater - INFO - Received signal 2 (SIGINT), stopping...
2019-11-21 10:37:08,968 - telegram.ext.updater - INFO - Received signal 2 (SIGINT), stopping...
2019-11-21 10:38:54,446 - telegram.ext.updater - INFO - Received signal 2 (SIGINT), stopping...
2019-11-21 10:42:41,383 - __main__ - CRITICAL - Shutdown initiated by Ben:Atkinson:1009602212
2019-11-21 10:42:43,123 - telegram.ext.updater - INFO - Received signal 2 (SIGINT), stopping...
2019-11-21 10:43:13,510 - __main__ - WARNING - Cannot establish group type private
2019-11-21 10:43:19,546 - telegram.ext.updater - INFO - Received signal 2 (SIGINT), stopping...
2019-11-21 10:43:31,602 - telegram.ext.updater - INFO - Received signal 2 (SIGINT), stopping...
2019-11-21 10:45:48,143 - telegram.ext.updater - INFO - Received signal 2 (SIGINT), stopping...
